spec/models/active_shipping_calculator_spec.rb

```diff
@@ -108,11 +106,11 @@ module ActiveShipping
 
     describe "compute" do
       it "should use the carrier supplied in the initializer" do
-        stub_request(:get, /http:\/\/production.shippingapis.com\/ShippingAPI.dll.*/).
-          to_return(:body => fixture(:normal_rates_request))
-        calculator.compute(package).should == 14.1
+        carrier.should_receive(:find_rates).and_return(ActiveShipping::RateResponse.new(true, "Foo"))
```

Comment:
It would be nice if we can start moving away from the older `should_receive` syntax to the newer `expect(foo).to receive`.